templates/CRM/Admin/Form/Setting/Miscellaneous.tpl

```diff
@@ -65,6 +65,13 @@
                 <p class="description">{ts 1="http://wkhtmltopdf.org/"}<a href="%1">wkhtmltopdf is an alternative utility for generating PDF's</a> which may provide better performance especially if you are generating a large number of PDF letters or receipts. Your system administrator will need to download and install this utility, and enter the executable path here.{/ts}</p>
             </td>
         </tr>
+        <tr class="crm-miscellaneous-form-block-remote_profile_submissions">
+          <td class="label">{$form.remote_profile_submissions.label}</td>
+          <td>{$form.remote_profile_submissions.html}<br />
+            <p class="description">{ts}If enabled, CiviCRM will allow users to submit profiles from external sites. This is disabled by default to limit abuse.{/ts}</p>
```

Comment:
I have no problem with this but just wonder if it should be in a .hlp file rather than here